app/services/project-skills-list.js

```diff
@@ -0,0 +1,58 @@
+import Ember from 'ember';
+import skillsList from 'code-corps-ember/utils/skills-list';
+
+const {
+  computed: {
+    alias
+  },
+  get,
+  getProperties,
+  inject: { service },
+  Service,
+  set
+} = Ember;
+
+export default Service.extend({
```

Comment:
I agree. If you'd like to create an issue, this PR is large enough as it is. There are a good number of cases here to use a mixin, and this will also be the case for the skill input when it comes time to use the keyboard events for things going forward.